time_capture/time_capture/doctype/time_capture/time_capture.py

```diff
@@ -132,6 +135,16 @@ def create_attendance(self):
 			attendance.custom_time_capture = self.name
 			attendance.save()
 
+	def validate_time_to_submit_in_days(self):
+		settings = frappe.get_single("Time Capture Settings")
```

Comment:
```suggestion
		settings = frappe.get_single("Time Capture Settings")
		days_to_submit = (
			frappe.db.get_value("Time Capture Settings", "time_to_submit_in_days")
			or 7
		)
```

Entsprechend unten auch anpassen.